### PR TITLE
New Change logs for next release

### PR DESCRIPTION
**Pull request by wu-sheng**
I am preparing the new change logs for beta. Welcome everyone to sync your upgrades for beta.

FYI @apache/skywalking-committers @candyleer @carlvine500 @liuhaoyang

This PR should stay in open until code frozen for beta release. Thanks everyone.

### CODE REVIEW

**Comment by candyleer** [comment]
UI changes 
Fix wrong startTime in trace view.

**Comment by wu-sheng** [comment]
> Fix wrong startTime in trace view.

@candyleer Done.

File: CHANGES.md [hunk 1]
```diff
@@ -1,19 +1,79 @@
...
+  - Add some test cases.
+ 
+#### UI Changes
+  - New trace query UI.
+  - New Application UI, merge server tab(removed) into applciation as sub page.
+  - Add Thermodynamic chart in overview page.
+  - Change all tps to cpm(calls per minutes).
+  - Fix wrong osName in server view.
+  - Fix wrong startTime in trace view.
+  - Fix some icons internet requirements.
```
Comment (hanahmily) [inline CHANGES.md]: Missing issues:
New Topology UI.
New response time / throughput TopN list.

Comment (wu-sheng) [reply CHANGES.md]: @hanahmily Updated.

**Pull request merged**